### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by lihuiliullh** [cvignac/DiGress#28]
May I know why you use the cross entropy between the clean graph and the noise graph as the training loss?
In paper like DDMP, it uses the distance between the true noise and the predicted noise. Why don't you use the same idea as DDMP, aka, the true graph noise and the predicted graph noise?

![image](https://user-images.githubusercontent.com/53407011/229307554-4a7b9f82-a6ac-495a-8b2b-b101217e4280.png)

![image](https://user-images.githubusercontent.com/53407011/229307564-99120b15-a46b-43e2-b70a-c1549cf5f14d.png)

**Comment by najwalb** [comment]
I believe this was simply a choice the authors made. I guess by DDMP you mean DDPM, a.k.a Ho et al. (2020)?
The paper actually explains that predicting the added noise or the clean graph are equivalent (at least with a Gaussian noise model). In this case, the authors went with the second option.

DiGress is following the model proposed in [D3PM](https://openreview.net/forum?id=h7-XixPCAL) (Austin et al. (2021)). You can find more discussion on the loss function of DiGress in this [issue](https://github.com/cvignac/DiGress/issues/5).

**Comment by cvignac** [comment]
Hi, thanks @najwalb for answering all these questions. As explained in the link above, predicting noise would be a strange choice in the case of discrete valued data. Furthermore, I believe that predicting the noise or predicting the clean data is fundamentally the same, although in some cases one may be slightly easier to train than the other.

Clement

**Comment by lihuiliullh** [comment]
Thanks, this has been a great help.
